spec/helpers/date.js

```diff
@@ -0,0 +1,40 @@
+const Lab = require('lab'),
+      lab = exports.lab = Lab.script(),
+      describe = lab.experiment,
+      it = lab.it,
+      testRunner = require('../spec-helpers').testRunner;
+
+
+describe('date/moment helper', function() {
+  const context = { date: '01/20/2019' };
```

Comment:
Just to make the tests less confusing (because we have both a context variable and a helper named `date`) can you rename this to something other than `date`? Maybe `today` or something?

Reply:
So, there will be only `moment` helper, so `date` shouldn't be confusing